.github/workflows/autoupdate-pre-commit-config.yml

```diff
@@ -17,10 +17,16 @@ jobs:
     steps:
       - name: Set up Python
         uses: actions/setup-python@v2
-      - uses: actions/cache@v2
+      - name: Cache multiple paths
+        uses: actions/cache@v2
         with:
-          path: ~/.cache/pre-commit
-          key: pre-commit-autoupdate
+          path: |
+            ~/.cache/pre-commit
+            ~/.cache/pip
```

Comment:
Going through the documentation, I found the following
https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python#specifying-a-python-version
![image](https://user-images.githubusercontent.com/5471162/96278879-ab47a180-0ff3-11eb-8d24-8be98bd85a09.png)
https://github.com/actions/setup-python#hosted-tool-cache
![image](https://user-images.githubusercontent.com/5471162/96278931-bac6ea80-0ff3-11eb-9f9c-a222f1212129.png)

`RUNNER_TOOL_CACHE/python` should take care of both `/opt/hostedtoolcache/Python/` and `c:\hostedtoolcache\windows\python\`

Reply:
Due to this comment:
> `setup-python` works by taking a specific version of Python or PyPy in this tools cache and adding it to PATH.

I would say we don't need to manually cache it at all.

Reply:
Here only `~/.cache/pre-commit` and `~/.cache/pip` make sense to me.

Reply:
Ok @s-weigand I'm removing all paths except the ones suggested by you although the packages installed/downloaded by the `actions/setup-python@v2` might not get cached by doing this